Classes/Hooks/RealUrlAutoConfiguration.php

```diff
@@ -0,0 +1,66 @@
+<?php
+declare(strict_types=1);
+namespace JWeiland\ServiceBw2\Hooks;
+
+/*
+ * This file is part of the service_bw2 project.
+ *
+ * It is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License, either version 2
+ * of the License, or any later version.
+ *
+ * For the full copyright and license information, please read the
+ * LICENSE.txt file that was distributed with this source code.
+ *
+ * The TYPO3 project - inspiring people to share!
+ */
+
+use DmitryDulepov\Realurl\Configuration\AutomaticConfigurator;
+use JWeiland\ServiceBw2\RealUrl\TitleMapping;
+
+/**
+ * Class RealUrlAutoConfiguration
+ *
+ * @package JWeiland\ServiceBw2\Hooks
```

Comment:
This will break our CI in future: https://github.com/jweiland-net/events2/blob/master/Build/.php_cs.php#L61

Reply:
Good hint. I´ll remove this from all classes inside the extension.